riot/lol/match_test.go

```diff
@@ -37,7 +38,12 @@ func TestMatchClient_List(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			client := internal.NewClient(api.RegionEuropeWest, "API_KEY", tt.doer, logrus.StandardLogger())
-			got, err := (&MatchClient{c: client}).List("id", 0, 1)
+			got, err := (&MatchClient{c: client}).List("id", 0, 1, &MatchListOptions{
+				Champion:  []int{1},
+				Queue:     []int{200},
+				BeginTime: time.Time{},
+				EndTime:   time.Time{},
```

Comment:
```suggestion
				BeginTime: time.Now(),
				EndTime:   time.Now(),
```
We should use actual values here for coverage.